packages/sources/vue/src/components/overlays/VtmnToast/VtmnToast.vue

```diff
@@ -0,0 +1,54 @@
+<script lang="ts">
+import '@vtmn/css-toast/dist/index-with-vars.css';
+import { reactive, computed, defineComponent, PropType } from 'vue';
+import { VtmnButton } from '../../index';
+
+export default /*#__PURE__*/ defineComponent({
+  name: 'VtmnToast',
+  inheritAttrs: false,
+  components: { VtmnButton },
+  props: {
+    withIcon: {
+      type: Boolean as PropType<Boolean>,
+      default: false,
+    },
+    withCloseButton: {
+      type: Boolean as PropType<Boolean>,
+      default: false,
+    },
+  },
+  emits: ['close'],
+  setup(props, { emit }) {
+    props = reactive(props);
+
+    const handleClose = (event: Event) => {
+      return emit('close', (event.target as HTMLInputElement).value);
+    };
+
+    return {
+      classes: computed(() => ({
+        'vtmn-toast': true,
+        show: true,
```

Comment:
```suggestion
        'show': true,
```

Reply:
Prettier automatically removes the quotes of the prop